src/test/regress/expected/minimal_cluster_management.out

```diff
@@ -20,6 +20,13 @@ SELECT 1 FROM master_add_node('localhost', :worker_1_port);
         1
 (1 row)
 
+-- make sure coordinator is always in metadata.
```

Comment:
I wonder if we ever DROP EXTENSION citus; in the tests, which would cause to lose this call.

So, to ensure all the tests run with this, I suggest we add a check to https://github.com/citusdata/citus/blob/main/src/test/regress/sql/check_mx.sql

`check_mx` is running at the end of almost all schedules (if not, we should add anyway)

The check could be as simple as:
```
SELECT count(*) > 1 as coordinator_exists  FROM pg_dist_node WHERE groupid = 0 AND isactive
```

And, we probably need to change the name of the file `check_mx` to `check_cluster_state` or such.

Reply:
When I check if the coordinator is in metadata at the end of a test schedule, I see that some tests in a schedule add/remove coordinator as a node. So at the end of the schedule run coordinator may not be in metadata since it is removed. 

Those tests need to be rewritten such that they do not add/remove coordinator as a node.
I am not sure how many tests need to be rewritten as of now. I will try to fix operations_schedule first.

Reply:
I'm fine merging this PR first, but still aiming to fix those cases one by one, and at the end, have `check_mx` cover coordinator in the metadata.

I think that'd be easier to progress?